docs/index-en.html

```diff
@@ -0,0 +1,4245 @@
+          <li> <a href="#participation" title="https://w3id.org/ontouml#participation">participation</a>
+          </li>
+          <li> <a href="#participational" title="https://w3id.org/ontouml#participational">participational</a>
+          </li>
+          <li> <a href="#phase" title="https://w3id.org/ontouml#phase">phase</a>
+          </li>
+          <li> <a href="#phaseMixin" title="https://w3id.org/ontouml#phaseMixin">phaseMixin</a>
+          </li>
+          <li> <a href="#quality" title="https://w3id.org/ontouml#quality">quality</a>
+          </li>
+          <li> <a href="#qualityNature" title="https://w3id.org/ontouml#qualityNature">qualityNature</a>
+          </li>
+          <li> <a href="#quantity" title="https://w3id.org/ontouml#quantity">quantity</a>
+          </li>
+          <li> <a href="#quantityNature" title="https://w3id.org/ontouml#quantityNature">quantityNature</a>
+          </li>
+          <li> <a href="#relator" title="https://w3id.org/ontouml#relator">relator</a>
+          </li>
+          <li> <a href="#relatorNature" title="https://w3id.org/ontouml#relatorNature">relatorNature</a>
+          </li>
+          <li> <a href="#role" title="https://w3id.org/ontouml#role">role</a> </li>
+          <li> <a href="#roleMixin" title="https://w3id.org/ontouml#roleMixin">roleMixin</a>
+          </li>
+          <li> <a href="#shared" title="https://w3id.org/ontouml#shared">shared</a>
+          </li>
+          <li> <a href="#situation" title="https://w3id.org/ontouml#situation">situation</a>
+          </li>
+          <li> <a href="#situationNature" title="https://w3id.org/ontouml#situationNature">situationNature</a>
+          </li>
+          <li> <a href="#subCollectionOf" title="https://w3id.org/ontouml#subCollectionOf">subCollectionOf</a>
+          </li>
+          <li> <a href="#subkind" title="https://w3id.org/ontouml#subkind">subkind</a>
+          </li>
+          <li> <a href="#subQuantityOf" title="https://w3id.org/ontouml#subQuantityOf">subQuantityOf</a>
+          </li>
+          <li> <a href="#termination" title="https://w3id.org/ontouml#termination">termination</a>
+          </li>
+          <li> <a href="#triggers" title="https://w3id.org/ontouml#triggers">triggers</a>
+          </li>
+          <li> <a href="#type" title="https://w3id.org/ontouml#type">type</a> </li>
+          <li> <a href="#typeNature" title="https://w3id.org/ontouml#typeNature">typeNature</a>
+          </li>
+        </ul>
+      </div>
+      <!--DESCRIPTION SECTION-->
+      <div id="description">
+        <h2 id="desc" class="list">OntoUML Metamodel Vocabulary: Description <span
+
+            class="backlink"> back to <a href="#toc">ToC</a></span></h2>
+        <span class="markdown"> This is a placeholder text for the description
+          of your ontology. The description should include an explanation and a
+          diagram explaining how the classes are related, examples of usage,
```

Comment:
Adding a diagram is a great idea to support documentation. I find it very helpful in DCAT and Ontolex.

Reply:
Could we replace the abstract syntax diagram with the one I created for the paper? I can create another one in the same style for the concrete syntax part.

Reply:
![metamodel-summary](https://github.com/OntoUML/ontouml-vocabulary/assets/2519935/2cc5197a-3213-4472-88e6-740cde071b83)